sources/bloodhound/src/bin/bottlerocket-checks/checks.rs

```diff
@@ -371,3 +371,43 @@ impl results::Checker for BR03010100Checker {
         }
     }
 }
+
+// =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<=
+
+pub struct BR03020100Checker {}
+
+impl results::Checker for BR03020100Checker {
+    fn execute(&self) -> results::CheckerResult {
+        let settings = vec![
+            "net.ipv4.conf.all.accept_source_route",
+            "net.ipv4.conf.default.accept_source_route",
+            "net.ipv6.conf.all.accept_source_route",
+            "net.ipv6.conf.default.accept_source_route",
+        ];
```

Comment:
Can you avoid the `vec!` macro here? (and in similar places). It seems like it is possible to do so since there are other places in this PR where a `[]` with a single element was passed to `check_output_contains!`

Reply:
Done!